### PR TITLE
edit to recommand-extensnion to receipt

### PR DESCRIPTION
**Pull request by readingsnail**
Multiful Repo is Good thing, but in my short thing, i not thinking 'not using >5%', so i re-write receipt.

### CODE REVIEW

File: docs/recipes/multiful-form-repo-fast-switching.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# multiful form repo Fast Switching n& maintain
```
Comment (riccardoferretti) [inline docs/recipes/multiful-form-repo-fast-switching.md:1]: I am not sure I understand what you mean by this sentence? 😅

Comment (readingsnail) [reply docs/recipes/multiful-form-repo-fast-switching.md]: Sorry, My Fault

Comment (riccardoferretti) [reply docs/recipes/multiful-form-repo-fast-switching.md]: what do you mean by "multiful form repo"?

Comment (ingalless) [reply docs/recipes/multiful-form-repo-fast-switching.md]: Just taking a guess:
Is it "multiple Foam repos"?

Comment (readingsnail) [reply docs/recipes/multiful-form-repo-fast-switching.md]: @ingalless yeah, sorry. @riccardoferretti Ingalless's description is right. English is my second language and almost not using in daily. only using internet things. (yeah native korean, almost people have phobia of english)

**Comment by riccardoferretti** [comment]
a couple of comments:
1. I see multiple files, but I think you only mean to have 1 recipe, right?
2. what do you think of calling the recipe something like "Quickly switch between multiple Foam repos" (and corresponding filename)?
3. "maintain for Multiful Form Repo" is not super clear in english (I reckon you are doing a literal translation from Korean), that's why I would avoid using that (also, I think you mean `Foam`, not `Form`)

Let me know what you think, thanks

File: docs/recipes/quickly-switch between-multiple-foam-repos.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Quickly switch between multiple Foam repos
+
+this #recipe is allow to Quickly switch between multiple Foam repos. and sorry for Setting/Content's Hangul the Korean Character.
+
+## Installation
+
+**This extension is not included in the template**
+
+This Method need [Project Manager](https://marketplace.visualstudio.com/items?itemName=alefragnani.project-manager) extension, so please install this extension.
```
Comment (lautjy) [inline docs/recipes/quickly-switch between-multiple-foam-repos.md:9]: Changing "need" to "needs the" makes the English more readable here.

File: docs/recipes/quickly-switch between-multiple-foam-repos.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Quickly switch between multiple Foam repos
+
+this #recipe is allow to Quickly switch between multiple Foam repos. and sorry for Setting/Content's Hangul the Korean Character.
```
Comment (lautjy) [inline docs/recipes/quickly-switch between-multiple-foam-repos.md:3]: Text is not Capitalized correctly:  "this" -> "This".  "and" -> "And".

Also not entirely sure what the sorry is about.. Could you just fix the issue instead?

File: docs/recipes/quickly-switch between-multiple-foam-repos.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+**This extension is not included in the template**
+
+This Method need [Project Manager](https://marketplace.visualstudio.com/items?itemName=alefragnani.project-manager) extension, so please install this extension.
+
+## Quickly switch between multiple Foam repos - Git Folder
+First, Need arrange Quickly switching Multiful Foam repos in one folder
+
+<img width="1322" alt="스크린샷 2021-06-02 오전 10 05 29" src="https://user-images.githubusercontent.com/1904967/120408485-1af37c80-c38a-11eb-8454-713a63ba5c7b.png">
+Second, In VSCode's Extensnion setting, select arrange folder, and extension setting, assign to this folder.
+```Project Manager › Git: Base Folders```
```
Comment (lautjy) [inline docs/recipes/quickly-switch between-multiple-foam-repos.md:16]: There's something wrong with your use of backtics. This does not render correctly in github.
You can see your version here:
https://github.com/readingsnail/foam/blob/master/docs/recipes/quickly-switch%20between-multiple-foam-repos.md

You could probably write:

```
Project Manager › Git: Base Folders
Indicates the base folders to search for Git projects
```

Though I am not sure what you are trying to achieve with a code block in this part of the text?

File: docs/recipes/quickly-switch between-multiple-foam-repos.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+```Indicates the base folders to search for Git projects```
+
+<img width="1322" alt="스크린샷 2021-06-02 오전 10 06 27" src="https://user-images.githubusercontent.com/1904967/120408569-41b1b300-c38a-11eb-8bc5-9c983600c0d8.png">
+or you can editing project manager's favorites. click to 'Edit Project' and editing ```Project.json``` like this
```
Comment (lautjy) [inline docs/recipes/quickly-switch between-multiple-foam-repos.md:20]: Also you backticks are wrong here. This doesn't render correctly at all.

You should use just 1 backtick for `code inside the text` and not 3 of them ````` . 

And when doing a code block, you can use 3 backticks, but they have to be on their own lines:

```

'''
code stuff
comes here
(And imagine that these 3 ' -marks are backticks for you. They are on their own lines with empty lines surrounding them, you see?
'''

```

File: docs/recipes/quickly-switch between-multiple-foam-repos.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+	{
+		"name": "Foam",
+		"rootPath": "/Users/hitchhiker/Foam",
+		"paths": [],
+		"group": "",
+		"enabled": true
+	},
+    
+This Part, **,** is need add to multiful Project. if end of project, deleted **,** and save the ```project.json```.
```
Comment (lautjy) [inline docs/recipes/quickly-switch between-multiple-foam-repos.md:29]: This line is not English text. Not sure what you are trying to say..

File: docs/recipes/quickly-switch between-multiple-foam-repos.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+		"rootPath": "/Users/hitchhiker/Foam",
+		"paths": [],
+		"group": "",
+		"enabled": true
+	},
+    
+This Part, **,** is need add to multiful Project. if end of project, deleted **,** and save the ```project.json```.  
+
+<img width="1120" alt="스크린샷 2021-06-02 오전 9 20 27" src="https://user-images.githubusercontent.com/1904967/120408615-5d1cbe00-c38a-11eb-9989-0f05f23bd1bb.png">
+That's it. if you need quickly switch between multiple Foam repos, click to left's folder icon or Git part.
```
Comment (lautjy) [inline docs/recipes/quickly-switch between-multiple-foam-repos.md:32]: Capitalization: "if" -> "If".
Also "left's folder" -> "left folder's" .

**Comment by riccardoferretti** [comment]
Thanks @readingsnail - I have taken your suggested extension and simply added it to the list of recommended extensions, this way it's easily accessible (and installation is described well enough in the extension's docs that I don't think we need an explicit recipe for it).
I have credited you in the commit (https://github.com/foambubble/foam/commit/dbb46369a20431075b2973f3fda317a1ebb7e152), thanks again!

**Pull request closed without merging**